akd_core/src/ecvrf/traits.rs

```diff
@@ -49,41 +49,35 @@ pub trait VRFKeyStorage: Clone + Sync + Send {
         self.get_vrf_private_key().await.map(|key| (&key).into())
     }
 
-    /// Returns the tree nodelabel that corresponds to a version of the uname argument.
-    /// The stale boolean here is to indicate whether we are getting the nodelabel for a fresh version,
+    /// Returns the [NodeLabel] that corresponds to a version of the label argument.
+    /// The stale boolean here is to indicate whether we are getting the [NodeLabel] for a fresh version,
     /// or a version that we are retiring.
     async fn get_node_label(
         &self,
-        uname: &AkdLabel,
+        label: &AkdLabel,
         stale: bool,
         version: u64,
     ) -> Result<NodeLabel, VrfError> {
-        let proof = self.get_label_proof(uname, stale, version).await?;
+        let proof = self.get_label_proof(label, stale, version).await?;
         let output: super::ecvrf_impl::Output = (&proof).into();
-        Ok(NodeLabel::new(output.to_truncated_bytes(), 256u32))
+        Ok(NodeLabel::new(output.to_truncated_bytes(), 256))
```

Comment:
nit: Can we just call `get_node_label_from_vrf_proof` here?

Reply:
Definitely... nice!